### ISSUE TITLE
UI enhancement in website

### ISSUE THREAD

**Issue by Vin205** [anuragverma108/SwapReads#971]
### Describe the feature

The website doesn't appear to be appealing to user as the styling, color contrast and the elements should be aligned properly to improve user experience. 

### Add ScreenShots

![IMG_20240528_230407](https://github.com/anuragverma108/SwapReads/assets/138381350/5611333d-32be-44a4-ad8a-af372828167b)
![IMG_20240528_230434](https://github.com/anuragverma108/SwapReads/assets/138381350/661a4675-6296-497c-bf06-b9459f1e7558)

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a GSSOC'24 contributor
- [X] I want to work on this issue

**Comment by anuragverma108** [comment]
Assigned!
Make sure to attach before & after screenshots in your PR. Also, star the repo before working on it. While creating your PR, ensure you pull all existing changes beforehand to avoid conflicts.
Happy open-source contributing!

**Comment by Praneeth-2602** [comment]
Since it is labeled "help wanted" can I also contribute? @anuragverma108
